samples/snippets/src/main/java/com/example/storage/object/CopyObject.java

```diff
@@ -34,14 +32,25 @@ public static void copyObject(
     // String objectName = "your-object-name";
 
     // The ID of the bucket to copy the object to
-    // String targetBucketName = "target-object-bucket"
+    // String targetBucketName = "target-object-bucket";
 
     Storage storage = StorageOptions.newBuilder().setProjectId(projectId).build().getService();
-    Blob blob = storage.get(sourceBucketName, objectName);
+    BlobId source = BlobId.of(sourceBucketName, objectName);
+    BlobId target =
+        BlobId.of(
+            targetBucketName, objectName); // you could change "objectName" to rename the object
```

Comment:
Any particular reason why you split it into multiple lines? Can we have it all in one line?

```java
BlobId target = BlobId.of(targetBucketName, objectName);
```

Reply:
It was originally written on one line, [Owlbot changed it](https://github.com/googleapis/java-storage/pull/1600/commits/4eded73d48a6c47afea6a6f16776f4d5cb048273#diff-de4371495edfc135eac9d6142b6a3c480b3e2593604c0d0452dd1fce28fa67d7R39) and will just change it back if i try to fix it